projects/plugins/jetpack/modules/masterbar/inline-help/class-inline-help.php

```diff
@@ -0,0 +1,47 @@
+<?php
+/**
+ * Inline Help
+ *
+ * Handles providing a LiveChat icon within WPAdmin until such time
+ * as the full live chat experience can be run in a non-Calypso environment.
+ *
+ * @package automattic/jetpack
+ */
+
+namespace Automattic\Jetpack\Dashboard_Customizations;
+
+use Automattic\Jetpack\Redirect;
+use Automattic\Jetpack\Status;
+
+/**
+ * Class Inline_Help.
+ */
+class Inline_Help {
+
+
+	public function __construct() {
+		$this->register_actions();
+	}
+
+	public function register_actions() {
+		add_action( 'admin_footer', array( $this, 'add_fab_icon' ) );
+
+		add_action( 'admin_enqueue_scripts', array( $this, 'add_fab_styles' ) );
+	}
+
+	public function add_fab_icon( $data ) {
+
+		if ( wp_doing_ajax() ) {
+			return $data;
```

Comment:
Actions don't need to return anything, that's something only required by filters.

Reply:
Whoops! Left this in as typo. Thanks for catching.

Reply:
Done